standards/alerting.md

```diff
@@ -27,7 +46,8 @@ The metric created from the metric filter MUST be used to trigger an alarm.
 
 2. Go to `CloudWatch` by clicking `Services` dropdown menu from the top navigation and choose `CloudWatch`.
 
-3. Go to `Logs` from the left navigation and find the log you want to build the meric on from the `Log Groups` list.
+3. Go to `Logs` from the left navigation and find the log group you want to build the metric on from the `Log Groups` list.
+   - Warning: Be careful about choosing the correct log group - especially for Elastic Beanstalk log groups (see a [previous post-mortem](../post-morterms/2018-03-30-sierra-api.md) that had metric filters improperly configured).
 
```

Comment:
Can we:

1. Change that folder name to "post-mortems", not "post-morterms".
1. Change this link back to 'post-mortems"

That "r" shouldn't be there.

Reply:
i wish the plural was posts mortem

Reply:
:-D

Reply:
I don't know why but I kept wanting to write "post-porte*r*ms"!

Anyway, thanks for pointing that out. I fixed the typos.